src/test/java/core/basesyntax/WorkWithFileTest.java

```diff
@@ -73,7 +73,6 @@ public void getStatisticAboutBanana() {
     @Test
     public void getStatisticAboutBananaRepeatedMethodCall() {
         workWithFile.getStatistic("banana.csv", BANANA_RESULT_FILE);
```

Comment:
never change the tests, should be recovered changes from src/test/java/core/basesyntax/WorkWithFileTest.java